perfkitbenchmarker/pkb.py

```diff
@@ -138,7 +138,12 @@
     'Type for all scratch disks. The default is standard')
 flags.DEFINE_integer('scratch_disk_iops', 1500,
                      'IOPS for Provisioned IOPS (SSD) volumes in AWS.')
-
+flags.DEFINE_integer('num_striped_drives', 1,
```

Comment:
We support benchmarks which use more than one data disk (e.g. copy_throughput). Specifying "num_striped_disks" in copy throughput will result in two striped disks each comprised of "num_striped_disks" disks, resulting in a total of 2 * "num_striped_disks" disks attached to the instance. I think "num_scratch_disks" would be more confusing than "num_striped_disks" in this case. Can you think of a better name (I don't particularly like "num_striped_disks" either and would be happy to replace it)?